From 6149bf54a04bb6d2109a78bb17f207167ce3182e Mon Sep 17 00:00:00 2001
From: Benda Xu <heroxbd@gentoo.org>
Date: Thu, 11 Sep 2014 18:20:44 +0900
Subject: disable class graph generation

Forwarded: no need
Description: The graphviz-generated class graph occupies 1GB disk space.
 It is too big to deal with.  We turn it off only to save space.
---
 doxygen.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doxygen.cfg b/doxygen.cfg
index 0fc2602..475aa67 100644
--- a/doxygen.cfg
+++ b/doxygen.cfg
@@ -34,7 +34,7 @@ PREDEFINED              = DOXYGEN_SHOULD_SKIP_THIS \
                           USE_THREADS \
 			  BOOST_PP_IS_ITERATING
 ALLEXTERNALS            = YES
-CLASS_GRAPH             = YES
+CLASS_GRAPH             = NO
 COLLABORATION_GRAPH     = YES
 GROUP_GRAPHS            = NO
 GRAPHICAL_HIERARCHY     = NO
